src/Code/201908_draft/Client/Schedule.js

```diff
@@ -72,9 +72,11 @@ const Schedule = class Schedule {
         `First parameter of Parameter "schedule.task" should be a non-empty string`,
       );
     }
+    const { canonical } = versioner(name);
     this.type = "task";
     this.name = name;
     this.input = input;
+    this.canonical = canonical;
     return this._processor.scheduleTask(this._getJob());
```

Comment:
It's probably a good idea to add version ?

Reply:
We don't have it on workflows either. If we add version, does it mean we are scheduling a specific version of the workflow? So if you provide a new version, you will have to cancel the schedule, and re-create a new one to use the new version?

Reply:
`version` is only a number.
`name` actually includes the versioning ("mywo_v1")
So the current implementation is: if you provide a canonical workflow, then the last version is used, if you provide a version, it will be used.

Providing the version here is only have the data if we need it